client/src/interface/Message.ts

```diff
@@ -0,0 +1,19 @@
+//for mock data UI testing, will change back to correct type with real data
+export interface Conversation {
+  conversationId: string;
+  imageURL: string | any;
+  fullName: string;
+  title: string;
+  createdAt: string;
+  messages: Message[];
+  lastMessageText: string;
+  isOnline: boolean;
+}
+
+export interface Message {
+  imageURL: string | null;
```

Comment:
Why are we using type null here and on the next line?

Reply:
I think we do not need  imageURL and fullName fields, but instead I will add messageId